a3fe/run/simulation.py

```diff
@@ -585,13 +550,41 @@ def update_paths(self, old_sub_path: str, new_sub_path: str) -> None:
             "crdfile": "somd.rst7",
         }
         for option, name in input_paths.items():
```

Comment:
If we avoid storing this information in the config, then we won't have to update here, and it will be cleaner.